### DISCUSSION TITLE
Can you replace mix with Vite

### DISCUSSION THREAD

**Discussion by animexxx** [tailpress/tailpress#176] [Ideas]
I have many heap crash like other issues opening. I've try Vite on Laravel and it work fast and better. Can you add Vite to this project.
Thanks for great start theme

**Comment by jeffreyvr** [comment]
Have you tried Laravel Mix instead? That's available now, Vite is not (yet) available for TailPress. Not sure if and when I get to supporting that.

You can use Mix by setting `--compiler="mix"` when you create a new theme with the installer.

Or grab the setup from here:
https://github.com/jeffreyvr/tailpress-installer/blob/master/stubs/mix/webpack.mix.js.stub
